integration_tests/cli/validator-demo.sh

```diff
@@ -177,6 +190,107 @@ check_response "$result" "\"pid\": $pid"
 check_response "$result" "\"productLabel\": \"$productName\""
 echo "$result"
 
+
+test_divider
+
+
+echo "node admin disables validator"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld tx validator disable-validator --from "$account" --yes")
+check_response "$result" "\"code\": 0"
+echo "$result"
+
+
+test_divider
+
+echo "Get a validator $address from disabled-validator query"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator disabled-validator --address=$address")
+check_response "$result" "\"approvals\": \[\]"
+check_response "$result" "\"disabledByNodeAdmin\": true"
+echo "$result"
+
+test_divider
+
+
+echo "node admin enables validator"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld tx validator enable-validator --from "$account" --yes")
+check_response "$result" "\"code\": 0"
+echo "$result"
+
+
+test_divider
+
+
+echo "Get a validator $address from disabled-validator query"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator disabled-validator --address="$address"")
+check_response "$result" "Not Found"
+echo "$result"
+
+
```

Comment:
Please check that a validator can be queried (`dcld query validator node`)

Reply:
Done! ✅